index.html

```diff
@@ -131,6 +131,12 @@ <h1 class="modal-title fs-5" id="advancedOptionsModalLabel">
         >
           日本語
         </button>
+        <button
+        class="btn btn-outline-secondary btn-sm"
```

Comment:
Lines 135 to 139 should have an additional two spaces of indentation.

Reply:
Thank you very much !
I try to fix the indentation, currently it should be OK now.